lib/rules/number-max-precision/__tests__/index.js

```diff
@@ -41,7 +41,7 @@ testRule({
 			code: "@IMPORT '1.123.css'",
 		},
 		{
-			code: 'a { background: url(1.123.jpg) }',
+			code: 'a { background: url(1.123.jpg) url("foo" 1.123.jpg) }',
 		},
 		{
 			code: 'a { my-string: "1.2345"; }',
```

Comment:
maybe replace this one by
```
font-family: "1.2345";
```
ref: https://drafts.csswg.org/css-fonts/#family-name-syntax

Reply:
The description for this test is :
> ignore all strings rather than only in 'content'

The purpose seems to have been to test a non-specific property with a string value.
In contrast with a known property where string values are expected.